controllers/alias.go

```diff
@@ -42,29 +42,26 @@ import (
 
 // ClusterReconciler reconciles a Cluster object.
 type ClusterReconciler struct {
-	Client                    client.Client
```

Comment:
night need to handle deprecation https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_cluster-api/10692/pull-cluster-api-apidiff-main/1795517221561176064